### PR TITLE
Update HU weight if scaleQty > huQty

### PR DESCRIPTION
**Pull request by cp-ps**
refs https://github.com/metasfresh/metasfresh/issues/14223

### CODE REVIEW

File: misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/step_scan/RawMaterialIssueStepScanComponent.jsx [hunk 1]
```diff
@@ -78,7 +83,7 @@ const RawMaterialIssueStepScanComponent = ({ wfProcessId, activityId, lineId, st
 
     const stepId = resolvedBarcodeData.stepId;
     const isWeightable = !!resolvedBarcodeData.isWeightable;
-    const isIssueWholeHU = !!resolvedBarcodeData.isIssueWholeHU;
+    const isIssueWholeHU = qty > resolvedBarcodeData.qtyHUCapacity;
```
Comment (teosarca) [inline misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/step_scan/RawMaterialIssueStepScanComponent.jsx:86]: stupid question, why not ">=" ?

File: misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/step_scan/computeStepScanPropsFromActivity.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import {
   getStepByIdFromLine,
 } from '../../../../../reducers/wfProcesses';
 
-export const computeStepScanPropsFromActivity = ({ activity, lineId, stepId }) => {
+export const computeStepScanPropsFromActivity = ({ activity, lineId, stepId, isProcessedQtyStillOnScale }) => {
```
Comment (teosarca) [inline misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/step_scan/computeStepScanPropsFromActivity.js:8]: what's the meaning / purpose of `isProcessedQtyStillOnScale` ?

what's `qtyAlreadyOnScale` and how that shall be used?

Comment (cp-ps) [reply misc/services/mobile-webui/mobile-webui-frontend/src/containers/activities/manufacturing/issue/step_scan/computeStepScanPropsFromActivity.js:8]: if that's true, it means the already issued qty is still on the scale, so the process needs to take that into consideration before issuing, i.e. qtyComingFromScale - qtyAlreadyIssued = qtyToIssueNow

**Comment by cp-ps** [comment]
will go to `uat` first

**Pull request closed without merging**